webrtc/libjingle/xmpp/boshsocket.cc

```diff
@@ -155,7 +156,15 @@ namespace buzz {
 #ifdef FEATURE_ENABLE_SSL
     if (tls_ != buzz::TLS_DISABLED) {
       first_socket = rtc::SSLAdapter::Create(first_socket);
+// For non-production builds, relax certificate requirements. This will allow the use
+// of a debugging proxy, for example.
+#ifndef _PRODUCT
```

Comment:
Not sure I follow your suggestion.

On Fri, Sep 18, 2015 at 1:29 PM, Will Kelleher <notifications@github.com>
wrote:

> In webrtc/libjingle/xmpp/boshsocket.cc
> <https://github.com/PersonifyInc/webrtc/pull/13#discussion_r39886022>:
>
> > @@ -155,7 +156,15 @@ namespace buzz {
> >  #ifdef FEATURE_ENABLE_SSL
> >      if (tls_ != buzz::TLS_DISABLED) {
> >        first_socket = rtc::SSLAdapter::Create(first_socket);
> > +// For non-production builds, relax certificate requirements. This will allow the use
> > +// of a debugging proxy, for example.
> > +#ifndef _PRODUCT
>
> Fair. We can possibly tweak the gclient.cmake file in StagePresence to do
> this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PersonifyInc/webrtc/pull/13/files#r39886022>.
>